bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/css/summernote.css

```diff
@@ -0,0 +1,16 @@
+.summernoteOverlay{
+	z-index:16777271;
+	background-color:black;
+	height:100%;
+	width:100%;
+	position: fixed;
+	top:0;
+	bottom:0;
+	left:0;
+	right:0;
+	-ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
+	filter: alpha(opacity=50);
+	-moz-opacity: 0.5;
+	-khtml-opacity: 0.5;
```

Comment:
According to http://caniuse.com/#search=opacity it seems we don't need `-moz` and `-khtml`.
May I nuke 'em?